app/models/provider.rb

```diff
@@ -31,6 +31,9 @@ class Provider < ApplicationRecord
   has_many :recommendations_uploads, class_name: "BulkUpdate::RecommendationsUpload"
   has_many :recommendations_upload_rows, class_name: "BulkUpdate::RecommendationsUploadRow", through: :recommendations_uploads
 
+  has_many :bulk_update_placements, class_name: "BulkUpdate::Placement"
+  has_many :placement_rows, class_name: "BulkUpdate::PlacementRow", through: :bulk_update_placements
+
```

Comment:
i'm slightly torn on this. on the one hand, it follows the same pattern as three lines above. on the other hand, i'm not convinced a model class really needs to know anything about bulk updates, and rather than accessing these as `provider.bulk_update_placements` we could have a query class `BulkUpdatePlacementsFor.call(provider:)` of a factory method in `BulkUpdate::Placement`. anyway, just a thought.